Makefile

```
@@ -17,6 +17,7 @@ CLI_PKG     = $(PROJECT_PKG)/cmd/oras
 GIT_COMMIT  = $(shell git rev-parse HEAD)
 GIT_TAG     = $(shell git describe --tags --abbrev=0 --exact-match 2>/dev/null)
 GIT_DIRTY   = $(shell test -n "`git status --porcelain`" && echo "dirty" || echo "clean")
+GO=go
```

I am assuming that you are managing Go installations according to https://go.dev/doc/manage-install.

I think it would be better to make the variable consistent:
```suggestion
GO_EXE     = go
```

---

Sure thing.

I'm running my old go builds for various reasons.

---

Done.